test/tests/api/v2_0/skupack_tests.py

```diff
@@ -244,13 +242,25 @@ def post_skupack(self):
                     Api().profiles_get_metadata_by_name('useless.json', scope=sku_id)
                     assert_equal(200, self.__client.last_response.status)
                     assert_equal(1, len(loads(self.__client.last_response.data)))
-
+                    
                     """Test DELETE:api/2.0/skus/:identifier/pack"""
+                    Api().skus_get()
+                    skus = loads(self.__client.last_response.data)
+                    LOG.info("Printing Sku from get skus before a delete sku pack")
+                    LOG.info(skus)
                     Api().skus_id_delete_pack(self.__packFolderId)
                     Api().skus_id_delete(self.__packFolderId)
```

Comment:
Need an additional GET after the pack delete to validate the sku pack related keys are correctly deleted from the sku object.